people-and-planet-ai/README.md

```diff
@@ -2,26 +2,40 @@
 
 ## 🦏 [Wildlife Insights -- _image-classification_](image-classification)
 
+[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/GoogleCloudPlatform/python-docs-samples/blob/master/people-and-planet-ai/image-classification/README.ipynb)
+
+[![Watch the video in YouTube](http://img.youtube.com/vi/hUzODH3uGg0/0.jpg)](https://youtu.be/hUzODH3uGg0)
```

Comment:
At first glance it seems like this is an image and not a video. But having this is super cool regardless! :)

Reply:
Thanks, I went ahead and added a text link as well so it's more explicit.